### PR TITLE
Include Rapidstart.md to run Spark, MLflow, and Auth integration end-to-end

### PR DESCRIPTION
**Pull request by dennyglee**
**PR Checklist**

- [X] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**
Include the rapidstart.md which is an extension to the UC quickstart.  It also includes instructions on how to utilize Spark, MLflow, and Auth integrations.

### CODE REVIEW

File: docs/rapidstart.md [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+adls.clientId.0=<ADLS_CLIENTID>
+adls.clientSecret.0=<SECRET>
+
+## GCS Storage Config (Multiple configs can be added by incrementing the index)
+gcs.bucketPath.0=<GCS_BUCKET>
+# Optional (If blank, it will use Default Application chain to find credentials)
+gcs.jsonKeyFilePath.0=/path/to/<SECRET>/gcp-key-uc-testing.json
+```
+
+### \[Optional\] Restart the UC Server
```
Comment (matthew-powers-db) [inline docs/rapidstart.md:56]: Not sure these backslashes are necessary, but shouldn't matter as long as the page renders properly

File: docs/rapidstart.md [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+marksheet_uniform
+numbers
+user_countries
+Time taken: 0.21 seconds, Fetched 4 row(s)
+```
+
+Let’s query the first five rows of the `marksheet` table.
+
+```sql
+SELECT * FROM default.marksheet LIMIT 5;
```
Comment (matthew-powers-db) [inline docs/rapidstart.md:166]: Does this work without specifying `catalog_name.schema_name.table_name`?

Comment (dennyglee) [reply docs/rapidstart.md:166]: Yes, on L78, the `bin/spark-sql` call denotes the catalog being used per ` --conf "spark.sql.defaultCatalog=unity"`

**Review by MrPowers — APPROVED** [review]
LGTM!

You'll need to add a line to the mkdocs.yml file if you'd like this to show up in the nav!

Love the deep dive 🤓

**Review by jamieknight-db — APPROVED** [review]
LGTM! One suggestion would be to add the 2 required dependencies for the UI (node and yarn) to the doc, but not sure where the appropriate place would be.

File: docs/rapidstart.md [hunk 3]
```diff
@@ -0,0 +1,717 @@
...
+* Granting the admin account as the metastore admin for the server
+
+
+## Testing User and Admin Authentication
+
+The following steps allow you to test user and admin authentication to your local Unity Catalog.
+
+### Use admin token to verify admin user is in local database
+
+As noted earlier, on startup the UC server configured itself and created the admin access token at `/etc/conf/token.txt`.  The following command uses the same admin authentication token to view the user list.
```
Comment (creechy) [inline docs/rapidstart.md:446]: nit: remove leading / in /etc/conf/token.txt

File: docs/rapidstart.md [hunk 4]
```diff
@@ -0,0 +1,717 @@
...
+
+### Add user account to the local database
+
+To fix this error, let’s start by adding your user account to the UC local database.  Use the CLI with the admin token to add your account to the local database.
+
+```bash
+bin/uc --auth_token $(cat etc/conf/token.txt) user create --name "Bobbie Draper" --email bobbie@rocinante
+```
+
+### Try to log in with your user account {#try-to-log-in-with-your-user-account}
```
Comment (creechy) [inline docs/rapidstart.md:500]: The ##try-to-log-in-with-your-user-account is being rendered.

**Review by creechy — APPROVED** [review]
LGTM - just a few nits and formatting.

**Review by kriscon-db — APPROVED** [review]
LGTM for models

File: docs/rapidstart.md (hunk 4 above)
Comment (dennyglee) [reply docs/rapidstart.md]: Thanks for catching this - fixed!

File: docs/rapidstart.md [hunk 5]
```diff
@@ -0,0 +1,717 @@
...
+bin/spark-sql --name "gcs-uc-test" \
+    --master "local[*]" \
+    --jars "https://repo1.maven.org/maven2/com/google/cloud/bigdataoss/gcs-connector/3.0.2/gcs-connector-3.0.2-shaded.jar" \
+    --packages "io.delta:delta-spark_2.12:3.2.1,io.unitycatalog:unitycatalog-spark_2.12:0.2.0" \
+    --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" \
+    --conf "spark.sql.catalog.spark_catalog=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.hadoop.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem" \
+    --conf "spark.hadoop.fs.AbstractFileSystem.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS" \
+    --conf "spark.hadoop.fs.gs.auth.type=ACCESS_TOKEN_PROVIDER" \
+    --conf "spark.hadoop.fs.gs.auth.access.token.provider=io.unitycatalog.spark.GcsVendedTokenProvider" \
```
Comment (bijanhoule) [inline docs/rapidstart.md:128]:
```suggestion
```

We can remove these properties now, as they were added to the connector in [#473](https://github.com/unitycatalog/unitycatalog/pull/473/files#diff-b496bfc650c9a1c585fb5709e010388ca74382670b419be5ebf4faa1bc499dd4R184)

**Review by bijanhoule — APPROVED** [review]
Thanks, @dennyglee !

File: docs/rapidstart.md (hunk 3 above)
Comment (dennyglee) [reply docs/rapidstart.md:444]: Ack!

**Comment by dennyglee** [comment]
Thanks for the suggestion, @jamieknight-db - the `quickstart` in the root `readme.md` contains the pre-requisites. Much appreciated!

**Pull request merged**